ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/card/card.jsp

```diff
@@ -26,6 +26,7 @@
                   java.util.ArrayList,
                   java.util.List" %>
 <%@ page import="java.util.Calendar" %>
+<%@ page import="java.util.Optional" %>
```

Comment:
(cosmetic) See all imports are united in one `import` tag but the last two. Not pretty)